src/assets/styles/components/playlist.scss

```diff
@@ -91,25 +91,25 @@
     }
 
     .cld-plw-panel-item {
-        
+
         margin-bottom: 4px;
-        
+
         &:last-child {
             margin-bottom: 0;
         }
 
     }
 }
 
-.cld-plw-item-title-next { 
+.cld-plw-item-title-next {
     font-weight: bold;
 }
 
 .cld-plw-item-info-wrap {
     position: absolute;
     bottom: 0;
     left: 0;
-    padding: 10px;
+    padding: 5% 0 5% 7%;
```

Comment:
@tsi  why %?

Reply:
Nothing super critical, could just as well use pixels but since you asked :) -
a. to be consistent with the [recommendations styles](https://github.com/cloudinary/cloudinary-video-player/blob/master/src/components/recommendations-overlay/recommendations-overlay.scss#L127).
b. since box height is not constant but relative to player width, it makes sense to use a relative unit.

Reply:
@tsi im not convinced, but we can stay with it for now